### PR TITLE
Policy issues

### PR DESCRIPTION
**Pull request by daniyalnawaz**

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "locals": {
-    "url": "https://data4america.org",
+    "url": "http://data4america.org",
```
Comment (dumbmatter) [inline config.json]: Why? We want https so we can take donations securely.

File: config.json [hunk 2]
```diff
@@ -12,7 +12,8 @@
   "require": {
     "moment": "moment",
     "_": "underscore",
-    "typogr": "typogr"
+    "typogr": "typogr",
+    "lo": "lodash"
```
Comment (dumbmatter) [inline config.json]: We really need underscore and lodash?

Comment (daniyalnawaz) [inline config.json]: You can remove lodash. I was using some of its functions in lifemap. Now
i've moved all its codebase to its own repo.

Comment (dumbmatter) [inline config.json]: Ok. Any comment on my other note?

Comment (daniyalnawaz) [inline config.json]: Yeah sorry... I think it got added during merge. You can remove the other
URL.

**Pull request merged**